ddp-workspace/tslint.json

```diff
@@ -135,7 +135,11 @@
     ],
     "space-before-function-paren": [
       true,
-      "never"
+      {
+        "anonymous": "never",
+        "asyncArrow": "always",
+        "named": "never"
+      }
```

Comment:
See https://palantir.github.io/tslint/rules/space-before-function-paren/ and https://stackoverflow.com/a/65579182.